notification2-examples/hello-world-microservice/src/main/resources/application.properties

```diff
@@ -1,7 +1,7 @@
 application.name=hello-notification
 server.port=8085
 
-example.source.id=29100
+example.source.id=883202
```

Comment:
what about removing this and instead loading just a first MO from the subscribed tenant, something like a:
```
@EventListener
    public void subscriptionAdded(MicroserviceSubscriptionAddedEvent event) {
        final ManagedObjectRepresentation mo = platform.getInventoryApi().getManagedObjectsByFilter(new InventoryFilter()).get().allPages().iterator().next();
...
```
this would just require to add `ROLE_INVENTORY_READ` into manifest

Reply:
thought: If @slc-softwareag suggestion is followed, my gut feel is that we should make sure it is not too hard for them to, alternatively, provide their own device Id if they want to

Reply:
I will avoid any change like this. Changes to the MS are not in the scope of this PR.